nss-day-cohort-48/giffygram-gopher
# Trashcan!

#### Changes Made
1. Modified file postlist.js to include trashcan and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cwj-trash-liked
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
When user clicks the trash can, it should delete the post above the img
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
